### PR TITLE
GG-54: steps-container styles, add steps-container to javascript quick…

### PR DESCRIPTION
**Pull request by mackarelfish**
added steps ui
image

### CODE REVIEW

**Comment by saikatmitra91** [comment]
Screenshot 2022-12-23 at 9 36 50 AM

the alignment is off @mackarelfish

File: styles/theme.css [hunk 1]
```diff
@@ -1370,4 +1370,45 @@ ul+p>img, ol+p>img {
...
+  margin-top: -3px;
+  margin-left: -41px;
+  width: 33px;
+  height: 33px;
+  justify-content: center;
+  align-items: center;
+  color: #999;
+  border-radius: 100%;
+  border: 4px solid #fff;
+  background: #f3f3f3;
```
Comment (saikatmitra91) [inline styles/theme.css:1401]: why not use css variables here ?

File: styles/theme.css [hunk 2]
```diff
@@ -1370,4 +1370,45 @@ ul+p>img, ol+p>img {
...
 li>p:nth-child(n+2) {
     margin-top: 16px;
+}
+
+.steps-container {
+  margin-left: 1rem;
+  padding-left: 1.5rem;
+  counter-reset: step;
+  border-left: 1px solid;
+  border-color: rgb(229 231 235/1);
```
Comment (saikatmitra91) [inline styles/theme.css:1380]: css variable ?

**Review by saikatmitra91 — CHANGES_REQUESTED** [review]

**Comment by mackarelfish** [comment]
@saikatmitra91 should be resolved by now

**Review by saikatmitra91 — APPROVED** [review]

**Pull request merged**